src/components/HistoryTables/MsigProposals/Proposal.tsx

```diff
@@ -80,24 +80,18 @@ export default function ProposalDetail(props: ProposalDetailProps) {
     return _msigTxsError
   }, [_msigTxsError, msigTxsLoading, proposal])
 
-  const actionRequired = useMemo(() => {
-    if (proposalFoundError) return false
-    if (!props.walletAddress?.robust && !props.walletAddress?.id) return false
-    if (!!proposal && !stateLoading && !stateError) {
-      const walletAddressIsSigner = isAddressSigner(
-        { id: props.walletAddress?.robust, robust: props.walletAddress?.id },
```

Comment:
Not sure why this was setting `{ id: ...robust, robust: ...id }`? But should be fine to shorten to `props.walletAddress`

Reply:
Wow.. good catch